presto-docs/src/main/sphinx/security/file-based-system-access-control.rst

```diff
@@ -0,0 +1,68 @@
+================================
+File Based System Access Control
+================================
+
+A system access control plugin enforces authorization at a global level,
+before any connector level authorization. You can provide your own
+implementation of ``SystemAccessControl``, as explained in
+:doc:`/develop/system-access-control`. Presto offers a built-in file based
+system access control plugin that supports specifying access control rules in a
+file.
+
+In order to enable this plugin, you must add an ``etc/access-control.properties``
+file containing two required properties, ``access-control.name`` which must be
+equal to ``file``, and ``security.config-file`` which must be set to the
+location of the config file. For example, if a config file named ``rules.json``
+resides in ``etc``, add an ``etc/access-control.properties`` with the following
+contents:
+
+.. code-block:: none
+
+   access-control.name=file
+   security.config-file=etc/rules.json
+
+The config file consists of a list of access control rules. These rules are
+matched in the order specified in the file. The user is
+granted the privileges from the first matching rule. All regular expressions
+default to ``.*`` if not specified.
+
+This plugin currently only supports catalog access control rules. If you want
+to limit access on a system level in any other way, you must implement a custom
+SystemAccessControl plugin (see :doc:`/develop/system-access-control`).
+
+Catalog Rules
+-------------
+
+These rules govern which catalogs particular users can access. The user is
+granted access to a catalog based on the first matching rule. If no rule
```

Comment:
The user...enabled. -- there is repetition here between this paragraph and two paragraphs up. Maybe make the paragraph two paragraphs up simpler:
"The config file consists of a list of access control rules in JSON format. These rules are matched in the order specified in the file. All regular expressiosn default to ``.*`` if not specified."